kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/ColumnMappingSuite.scala

```diff
@@ -124,6 +125,91 @@ class ColumnMappingSuite extends AnyFunSuite {
       .isEqualTo(15)
   }
 
+  test("finding max column id with nested struct type") {
+    val nestedStruct = new StructType()
+      .add("d", IntegerType.INTEGER,
+        FieldMetadata.builder.putLong(COLUMN_MAPPING_ID_KEY, 3).build())
```

Comment:
Can we move `FieldMetadata.builder.putLong(COLUMN_MAPPING_ID_KEY, 3).build()` to a succinct helper method?

`fieldMetadataColMapId(x)`

Reply:
done